GVFS/GVFS.Service/GVFSService.Mac.cs

```diff
@@ -16,13 +20,32 @@ public class GVFSService
         private Thread serviceThread;
         private ManualResetEvent serviceStopped;
         private string serviceName;
+        private RepoRegistry repoRegistry;
+        private string serviceDataLocation;

```

Comment:
Doesn't look like we need this member variable since it is only used in the constructor.